src/wild_encounter.c

```diff
@@ -1134,3 +1134,24 @@ bool8 StandardWildEncounter_Debug(void)
     DoStandardWildBattle_Debug();
     return TRUE;
 }
+
+u8 ChooseHiddenMonIndex(void)
+{
+    #ifdef ENCOUNTER_CHANCE_HIDDEN_MONS_TOTAL
```

Comment:
Regular expression rather than preproc

Reply:
Pretty sure it needs to stay preproc'ed because it doesn't exist until a user adds at least one set of hidden mon data to wild_encounters.json